client/src/components/Landing.js

```diff
@@ -24,15 +24,15 @@ function Landing() {
         </div>
         <div className="column small-12 large-4">
           <h1>1</h1>
-          <p className="text-left">Activity Director Schedules Open Spots for Musicians</p>
+          <img src="../../public/img/how/how_organizer.png" alt="Organizers Create Volunteering Spots" />
```

Comment:
I'm going to go ahead and merge this because it looks good anyway, even if I don't understand it.